docs/gpa.md

```diff
@@ -233,78 +231,23 @@ The information included in the notice of intended procurement covers all the in
     <tbody>
       <tr class="section">
         <td>X:7</td>
-        <td colspan="2">A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
-      </tr>
-      <tr>
-        <td>X:7(a)</td>
-        <td>the procurement, including the nature and the quantity of the goods or services to be procured or, where the quantity is not known, the estimated quantity and any requirements to be fulfilled, including any technical specifications, conformity assessment certification, plans, drawings or instructional materials;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(b)</td>
-        <td>any conditions for participation of suppliers, including a list of information and documents that suppliers are required to submit in connection with the conditions for participation;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(c)</td>
-        <td>all evaluation criteria the entity will apply in the awarding of the contract, and, except where price is the sole criterion, the relative importance of such criteria;</td>
-        <td markdown=1>
-
-* 
-</td>
-      </tr>
-      <tr>
-        <td>X:7(d)</td>
-        <td>where the procuring entity will conduct the procurement by electronic means, any authentication and encryption requirements or other requirements related to the submission of information by electronic means;</td>
```

Comment:
We can add 'electronicSubmission' to `tender.submissionMethod`, map/append `tender.submissionMethodDetails`, and mention `tender.communication.atypicalToolUrl` from the [communication](https://extensions.open-contracting.org/en/extensions/communication/master/) extension, in case they have such information.

Reply:
Done in ba57af968e0d